### ISSUE TITLE
[Visual Bug] Water in a boats

### ISSUE THREAD

**Issue by juraj-hrivnak** [Meldexun/EntityCulling#10]
**Describe the bug**
Boats have water in them.

**To Reproduce**
Install Entity Culling and that's it.

**Expected behaviour**
No water in boats

**Screenshots**
`Without Entity Culling:`
![2021-02-09_19 40 00](https://user-images.githubusercontent.com/71150936/107412142-a0b79780-6b0f-11eb-99e1-156c14a24b55.png)
`With Entity Culling:`
![2021-02-09_19 45 21](https://user-images.githubusercontent.com/71150936/107412154-a1e8c480-6b0f-11eb-8c26-e1a55587c9f6.png)

**Versions**
Entity Culling: 2.0.8
Forge: 14.23.5.2854
Minecraft: 1.12.2

**Other**
Latest log: https://paste.ee/p/XKL0i
Singleplayer/Multiplayer: **Both**
Also tested that no other mods are required.

**Comment by Meldexun** [comment]
I unable to reproduce this with only entity culling

**Comment by Meldexun** [comment]
Nvm. I messed up my run configuration.

**Comment by juraj-hrivnak** [comment]
Hmm. I don't know. I am using MultiMC
![2021-02-10_17 17 17](https://user-images.githubusercontent.com/71150936/107538166-f3eb2200-6bc3-11eb-85da-7ce0e2b66b07.png)

These are my specs:
```
Minecraft Version: 1.12.2
	Operating System: Windows 8.1 (amd64) version 6.3
	Java Version: 1.8.0_281, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 2058299928 bytes (1962 MB) / 2272788480 bytes (2167 MB) up to 3817865216 bytes (3641 MB)
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms2048m -Xmx4096m
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: 
	Loaded coremods (and transformers): 
EntityCullingPlugin (EntityCulling-1.12.2-2.0.8.jar)
  meldexun.entityculling.plugin.ClassTransformer
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 391.35' Renderer: 'GeForce GT 620/PCIe/SSE2'
```
But I was experiencing this bug also on a friends computer.

**Comment by Meldexun** [comment]
Fixed in v1.12.2-2.0.9
